airbyte-server/src/main/java/io/airbyte/server/handlers/helpers/ConnectionScheduleHelper.java

```diff
@@ -44,6 +50,11 @@ public static void populateSyncFromScheduleTypeAndData(final StandardSync standa
                 new BasicSchedule().withTimeUnit(ApiPojoConverters.toBasicScheduleTimeUnit(scheduleData.getBasicSchedule().getTimeUnit()))
                     .withUnits(scheduleData.getBasicSchedule().getUnits())))
             .withManual(false);
+
+        // populate legacy `schedule` column until it's removed.
+        standardSync.withSchedule(new Schedule()
```

Comment:
for my understanding, why populate this?

Reply:
I talked about this with @mfsiega-airbyte a bit, right now we still have some places in code that read the legacy column. Our core temporal workflow code doesn't, but some extraneous places do, like [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-scheduler/scheduler-persistence/src/main/java/io/airbyte/scheduler/persistence/job_tracker/TrackingMetadata.java#L41) for example.

I didn't want to expand the scope of this PR too much by also trying to eliminate every place in our code that reads from the column. We're still relying on "dual-write" for schedules, so this line makes sure that we still "dual-write" to the legacy column even for patch requests that don't specify the `schedule` property.

Reply:
ok!